apiserver/src/test/java/com/okbut/api/books/controller/BookControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package com.okbut.api.books.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.okbut.api.books.application.BookService;
+import com.okbut.api.books.domain.Book;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.restdocs.AutoConfigureRestDocs;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.LocalDateTime;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.springframework.restdocs.mockmvc.RestDocumentationRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ActiveProfiles("test")
+@DisplayName("BookController 클래스")
+@WebMvcTest(BookController.class)
+@AutoConfigureRestDocs
+class BookControllerTest {
+
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private BookService bookService;
+
+    @BeforeEach
+    void setUp() {
+        objectMapper = new ObjectMapper();
+    }
+
+    @Nested
+    @DisplayName("POST /books 요청은")
+    class Describe_post_books {
+        final Long isbn = 1234567890123L;
+        final Book book =
+                Book.builder().isbn(isbn).name("도서명").publisher("출판사").author("테드창")
+                        .editor("편집자").language("ko").publicationDtm(LocalDateTime.now())
+                        .regDtm(LocalDateTime.now()).build();
+
+        @BeforeEach
+        void mocking() {
+            given(bookService.registerBook(any(Book.class))).willReturn(book);
+        }
+
+        // TODO : 만약 관리자 권한을 가진 아이디가 주어진다면 (관리자 권한은 뭘로 판별하지..)
+
+        @Test
+        @DisplayName("도서를 등록하고 응답한다.")
+        void it_register_a_book_and_returns() throws Exception {
+            var request = post("/books")
+                    .accept(MediaType.APPLICATION_JSON)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(objectMapper.writeValueAsString(book));
+
+            mockMvc.perform(request)
+                    .andExpect(status().isCreated())
+                    .andExpect(jsonPath("$.[?(@.isbn == '%d')]", isbn).exists())
```

Comment:
이렇게 변경하는 것이 더 읽기 쉽지 않을까요?

```suggestion
                    .andExpect(jsonPath("$.isbn", equalTo(isbn)))
```